<ui version="4.0" >
 <author>Sebastian Trueg</author>
 <class>NepomukConfigWidget</class>
 <widget class="QWidget" name="NepomukConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>448</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Basic Settings</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Strigi Desktop Search</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Strigi desktop search allows to search files by content instead of just by name.</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="m_checkEnableStrigi" >
              <property name="text" >
               <string>Enable Strigi Desktop Search</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="KLed" name="m_strigiStatus" />
              </item>
              <item>
               <widget class="QLabel" name="m_strigiStatusLabel" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Strigi not running</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Nepomuk Semantic Desktop</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Nepomuk Semantic Desktop enables tagging and rating of files integrated with the Strigi Desktop Search.</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_checkEnableNepomuk" >
            <property name="text" >
             <string>Enable Nepomuk Semantic Desktop</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>21</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Advanced Settings</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="KEditListBox" name="m_editStrigiFolders" >
         <property name="toolTip" >
          <string>The folders Strigi scans for indexable files</string>
         </property>
         <property name="whatsThis" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select the folders that Strigi should scan for files to index. The files in these folders will be searchable through Strigi Desktop Search.&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="title" >
          <string>Strigi Index Folders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KEditListBox" name="m_editStrigiExcludeFilters" >
         <property name="toolTip" >
          <string>Filters which define the files that Strigi should ignore</string>
         </property>
         <property name="whatsThis" >
          <string/>
         </property>
         <property name="title" >
          <string>Strigi Index Exclude Filters</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>371</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
